README.md

```diff
@@ -13,73 +10,66 @@
+- And various useful functions:
+  - toggle repo visibility;
+  - shortcut to input repo;
+  - share on twitter quickly;
+  - support input multiple repos;
+  - ...wait for you finding out
 
-![web-demo](https://raw.githubusercontent.com/bytebase/star-history/master/public/star-history.gif)
+## 🌠 Screenshots
 
-## [As an extension](https://chrome.google.com/webstore/detail/star-history/iijibbcdddbhokfepbblglfgdglnccfn)
+<a href="https://star-history.com"><img width="800px" src="https://user-images.githubusercontent.com/24653555/154391264-312b448b-f851-41bf-bb8d-4c21ec6795b6.gif" />
+</a>
 
-![extension-demo](https://raw.githubusercontent.com/timqian/images/master/star-history-extension.gif)
+### [🧩 Also available as free chrome extension](https://chrome.google.com/webstore/detail/star-history/iijibbcdddbhokfepbblglfgdglnccfn)
 
-> Note: You can [load the `./extension` folder to chrome](https://superuser.com/a/247654) to install the extension too.
+<a href="https://chrome.google.com/webstore/detail/star-history/iijibbcdddbhokfepbblglfgdglnccfn"><img width="800px" src="https://user-images.githubusercontent.com/24653555/154391326-61b65d8f-3f9f-4432-b773-5988be75b0ea.png" /></a>
 
-## Access Token
+## 🏗 Development
 
-[star-history.com](https://star-history.com) uses GitHub API to retrieve repository metadata. When user exceed the rate [limit of unauthenticated requests](https://developer.github.com/v3/#rate-limiting). Star history will need your personal access token to unlimit it.
+star-history is built with a modern tech stack: Vue + Vite + TailwindCSS.
 
-If you don't already have one, [create one](https://github.com/settings/tokens/new), and add to star-history (no scope to your personal data is needed)
+### Prerequisites
 
-## Develop
+- [Node.js](https://nodejs.org/en/download/)
+- [Yarn](https://yarnpkg.com/)
 
-### Website
+### Install dependencies
 
-```bash
-npm run startWebsite
+```shell
+yarn
 ```
 
-### Extension
-
-```bash
-npm run buildExtension
-# load the extension folder as unpacked extension into chrome to view it
-```
-
-## Build and Deploy
-
-### Website
-
-```bash
-# deploy to star-history.com
-npm run deployWebsite
-```
-
-### Extension
-
-```bash
-npm run buildExtension
-# zip extension folder and publish to chrome web store
-```
-
-## Updates
-
-- 2019-8-28: use [chart.xkcd](https://github.com/timqian/chart.xkcd) to plot the graph
+### Start with your interest
 
-- 2019-3-06: Add personal access token; update style; mono repo
+- **Main website** is the homepage of star-history with most of useful features and blogs about open source.
 
-- 2016-6-30: Alert to notie
+  ```shell
+  yarn dev
+  ```
 
-- 2016-6-28: Add clear btn
+  The website will be served at http://localhost:3000.
 
-- 2016-6-28: Better view for "many star" repos (use current star number as the last point on the graph)
+- **Chrome extension** supports the basic chart viewer as a free additional product.
```

Comment:
I still suggest to put chrome extension on the top, or at least mention it with a #link pointing to this section